.buildkite/scripts/steps/functional/performance_playwright.sh

```diff
@@ -39,8 +39,13 @@ if [ "$BUILDKITE_PIPELINE_SLUG" == "kibana-performance-data-set-extraction" ]; t
   node scripts/run_performance.js --kibana-install-dir "$KIBANA_BUILD_LOCATION" --skip-warmup
 else
   # pipeline should use bare metal static worker
-  echo "--- Running performance tests"
-  node scripts/run_performance.js --kibana-install-dir "$KIBANA_BUILD_LOCATION"
+  if [[ -z "${JOURNEYS_GROUP+x}" ]]; then
+    echo "--- Running performance tests"
+    node scripts/run_performance.js --kibana-install-dir "$KIBANA_BUILD_LOCATION"
```

Comment:
@delanni it looks like empty is ok, based on this line right?

Reply:
I misread the condition, it is in fact ok like it is

Reply:
Meaning, an empty or missing group would end up in the first branch, where we run everything

Reply:
yes, exactly. Thank you both for double check

Reply:
Yeah this was a good nerd-scussion!  ❤️ hahahaa